app/views/rubygems/_aside.html.erb

```diff
@@ -15,6 +15,15 @@
       <span class="gem__downloads"><%= number_with_delimiter(@latest_version.downloads_count) %></span>
     </h2>
   </div>
+
+
+  <h2 class="gem__ruby-version__heading t-list__heading">
+    <%= t('.gem_version_age') %>:
+    <span class="gem__rubygem-version-age">
+      <p><%= time_ago_in_words(@latest_version.authored_at) %> ago</p>
```

Comment:
@segiddins if I understand it well, this is going to break caching, since the page content will change in time even with no resource update. Would it be better to use JS solution like https://momentjs.com/docs/#/displaying/fromnow/?

Reply:
I have found out also https://github.com/basecamp/local_time.

Reply:
@robbyrussell would you mind to try to use [local_time](https://rubygems.org/gems/local_time) gem? Feel free to ping me if you need help.

Reply:
I'm sorry for not considering the caching aspect that is in production.

The install instructions for the current releases of local_time assume you might have importmaps and/or webpacker in place. Are there plans for this project to use either approach going forward? (and past versions on NPM)

Reply:
No, but can't you just import one of those (https://github.com/basecamp/local_time/tree/main/app/assets/javascripts) using current pipeline (require here - https://github.com/rubygems/rubygems.org/blob/master/app/assets/javascripts/application.js)?

Reply:
@robbyrussell there is PR introducing importmaps at https://github.com/rubygems/rubygems.org/pull/4392. Maybe it is worth to wait for that one to land before tackling this.